### PR TITLE
ICMSLST-648 Fix update request for ILB Admin

### PR DESCRIPTION
**Pull request by kevinetienne**
- Remove non ILB Admin views for Update Requests (To be implemented in ICMLST-679)
 - Add missing template to create Update Request
 - Fix closing request
 - Move update requests urls from import to case
 - Make update request available to export case management (ICMSLST-432)

### CODE REVIEW

File: web/domains/case/urls.py [hunk 1]
```diff
@@ -81,6 +81,11 @@
     ),
 ]
 
+update_requests_urls = [
+    path("", views.manage_update_requests, name="manage-update-requests"),
```
Comment (oskusalerma) [inline web/domains/case/urls.py:85]: can this url be "manage/", we don't have any other "empty" urls like this here

File: web/domains/case/urls.py [hunk 2]
```diff
@@ -81,6 +81,11 @@
...
+    path("<int:update_request_pk>/close/", views.close_update_request, name="close-update-request"),
```
Comment (oskusalerma) [inline web/domains/case/urls.py:86]: if we're going to have respond-update-request etc in the future, could you add this under its own path already that captures the pk?

File: web/domains/case/views.py [hunk 3]
```diff
@@ -420,16 +420,52 @@ def archive_withdrawal(
...
+        elif case_type == "export":
+            template_code = "CA_APPLICATION_UPDATE_EMAIL"
+
+            # TODO: replace with case reference ICMSLST-348
+            placeholder_content = {
+                "CASE_REFERENCE": application_pk,
+                "EXPORTER_NAME": application.exporter.name,
+                "CASE_OFFICER_NAME": request.user,
+            }
+        else:
```
Comment (oskusalerma) [inline web/domains/case/views.py:461]: line 428 will already have errored out if case_type is not in import/export, this is not needed

File: web/domains/case/views.py [hunk 4]
```diff
@@ -439,7 +475,7 @@ def _manage_update_requests(
                 update_request.status = models.UpdateRequest.OPEN
                 update_request.save()
 
-                application.status = model_class.UPDATE_REQUESTED
+                application.status = ImportApplication.UPDATE_REQUESTED
```
Comment (oskusalerma) [inline web/domains/case/views.py:478]: this seems like a backwards change...? why would export application use an importapplication constant?

File: web/domains/case/views.py [hunk 5]
```diff
@@ -449,8 +485,22 @@ def _manage_update_requests(
...
 
+                if case_type == "import":
+                    contacts = get_users_with_perms(
+                        application.importer, only_with_perms_in=["is_contact_of_importer"]
+                    ).filter(user_permissions__codename="importer_access")
+                elif case_type == "export":
+                    contacts = get_users_with_perms(
+                        application.exporter, only_with_perms_in=["is_contact_of_exporter"]
+                    ).filter(user_permissions__codename="exporter_access")
+                else:
```
Comment (oskusalerma) [inline web/domains/case/views.py:496]: same, case_type has already been checked to be import/export

File: web/domains/case/views.py [hunk 6]
```diff
@@ -463,14 +513,14 @@ def _manage_update_requests(
...
-            "process_template": f"web/domains/case/{url_namespace}/partials/process.html",
+            "process_template": f"web/domains/case/{case_type}/partials/process.html",
             "process": application,
             "task": task,
             "page_title": f"{application.application_type.get_type_description()} - Update Requests",
             "form": form,
             "update_requests": update_requests,
             "current_update_request": current_update_request,
-            "url_namespace": url_namespace,
+            "case_type": case_type,  # TODO: ICMSLST-670 remove
```
Comment (oskusalerma) [inline web/domains/case/views.py:523]: umm...no. why would we ever remove case_type? this comment needs to go on url_namespace=case_type line. or if you've tested and this all works without url_namespace then you don't need any comment at all, and neither do you need url_namespace

File: web/domains/case/views.py [hunk 7]
```diff
@@ -463,14 +513,14 @@ def _manage_update_requests(
             )
 
         context = {
-            "process_template": f"web/domains/case/{url_namespace}/partials/process.html",
+            "process_template": f"web/domains/case/{case_type}/partials/process.html",
```
Comment (oskusalerma) [inline web/domains/case/views.py:516]: have you checked you need this anymore? a 1-minute browse through your templates seems to indicate to me you don't, but please try it out

**Review by oskusalerma — CHANGES_REQUESTED** [review]

File: web/domains/case/views.py (hunk 7 above)
Comment (kevinetienne) [reply web/domains/case/views.py:516]: I confirm it's defined in the base template for case management https://github.com/uktrade/icms/blob/43577b3d64357e768652552f6bc685bc583fae67/web/templates/web/domains/case/manage/base.html#L6

File: web/domains/case/urls.py (hunk 1 above)
Comment (kevinetienne) [reply web/domains/case/urls.py]: good idea fixed in https://github.com/uktrade/icms/pull/392/commits/c808497089ee578135b4f2fbca308193e2bfb673

File: web/domains/case/urls.py (hunk 2 above)
Comment (kevinetienne) [reply web/domains/case/urls.py]: fixed in https://github.com/uktrade/icms/pull/392/commits/c808497089ee578135b4f2fbca308193e2bfb673 let me know if this wasn't what you meant.

File: web/domains/case/views.py (hunk 4 above)
Comment (kevinetienne) [reply web/domains/case/views.py]: fixed in https://github.com/uktrade/icms/pull/392/commits/c808497089ee578135b4f2fbca308193e2bfb673, there was another ImportApplication laying a few lines above too, thanks

File: web/domains/case/views.py (hunk 5 above)
Comment (kevinetienne) [reply web/domains/case/views.py]: fixed in https://github.com/uktrade/icms/pull/392/commits/c808497089ee578135b4f2fbca308193e2bfb673

File: web/domains/case/views.py (hunk 7 above)
Comment (kevinetienne) [reply web/domains/case/views.py]: fixed in https://github.com/uktrade/icms/pull/392/commits/c808497089ee578135b4f2fbca308193e2bfb673

**Review by oskusalerma — APPROVED** [review]

**Pull request merged**